python/lsst/cp/pipe/ptc.py

```diff
@@ -139,7 +139,7 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
     maskNameList = pexConfig.ListField(
         dtype=str,
         doc="Mask list to exclude from statistics calculations.",
-        default=['DETECTED', 'BAD', 'NO_DATA'],
+        default=['SUSPECT', 'BAD', 'NO_DATA'],
```

Comment:
Seems like adding SUSPECT is 👍 but why are we removing DETECTED? Maybe it should never have been there in the first place? Seems weird that detection would even have been run, so I don't disagree with this change, just curious what's happening here.

Reply:
I also think that perhaps this should not have been there in the first place (it does not seem to me that DETECTED pixels should be in the list of pixels to mask), so I considered removing it.